citations-tool/tool/src/webapp/css/citations.css

```diff
@@ -173,10 +173,17 @@ ol.vertical li.placeholder::before {
 .descriptionButtons {
 	padding:5px;
 }
+ol.serialization li div.sectionEditor:hover {
+	cursor: pointer;
+	background: #2a4667 none repeat scroll 0 0;
```

Comment:
There isn't any inline-style equivalent for :hover I don't think.  I could do something clever in the js...